src/Microsoft.NetCore.Analyzers/Core/Security/SourceTriggeredTaintedDataAnalyzerBase.cs

```diff
@@ -84,6 +84,18 @@ public override void Initialize(AnalysisContext context)
                                 },
                                 OperationKind.Invocation);
 
+                            operationBlockStartContext.RegisterOperationAction(
+                                operationAnalysisContext =>
+                                {
+                                    var arrayInitializerOperation = (IArrayInitializerOperation)operationAnalysisContext.Operation;
```

Comment:
>var [](start = 36, length = 3)

Nitpick: Be consistent with the style that already exists in a file.  In this case, use IArrayInitializerOperation.